tutorials/shaders/shader_reference/particle_shader.rst

```diff
@@ -157,7 +158,7 @@ Process built-ins
 +------------------------------------+-----------------------------------------------------------------------------------------------------+
 | in bool **COLLIDED**               | ``true`` when the particle has collided with a particle collider.                                   |
 +------------------------------------+-----------------------------------------------------------------------------------------------------+
-| in vec3 **COLLISION_NORMAL**       | A normal of the last collision. If there is no collision detected it is equal to ``vec3(0.0)``.     |
+| in vec3 **COLLISION_NORMAL**       | A normal of the last collision. If there is no collision detected it is equal to ``(0.0,0.0,0.0)``. |
```

Comment:
```suggestion
| in vec3 **COLLISION_NORMAL**       | A normal of the last collision. If there is no collision detected it is equal to ``(0.0,0.0,0.0)``. |
```
```suggestion
| in vec3 **COLLISION_NORMAL**       | A normal of the last collision. If there is no collision detected it is equal to ``(0.0, 0.0, 0.0)``. |
```
And adjust the whole table

Reply:
I prefer `vec3(0.0, 0.0, 0.0)` for the literal, to make the type clear. I'll make sure to use the space between x, y, z, w for any vector literals in these pages though.

Reply:
Actually, looking at the pages again, you're right that just ``(0.0, 0.0, 0.0)`` is better. In other cases, writing the vector literals without the `vec3`, `vec2` constructor looks better, and in this case it could go either way. So I'll default to no constructor.